internal/apikey/apikey.go

```diff
@@ -49,13 +52,13 @@ func NewTkApiKey() (*ApiKey, error) {
 
 // Encode an ECDSA private key into the Turnkey format
 // For now, "Turnkey format" = raw DER form
-func EncodePrivateKey(privateKey *ecdsa.PrivateKey) string {
-	return fmt.Sprintf("%064x", privateKey.D)
+func EncodePrivateKey(privateKey *ecdsa.PrivateKey) []byte {
+	return []byte(fmt.Sprintf("%064x", privateKey.D))
 }
```

Comment:
I think this is a bit misleading because the bytes returned aren't the bytes of the private key. They're the bytes of the base64 string representation of the private key. Keeping the type as "string" makes it clear that we're not dealing with private key bytes IMO.

Reply:
Ah!  It was not at all clear, looking at this that that was what we were working with... in that case, I must agree.